.gitignore

```diff
@@ -8,3 +8,5 @@
 /build
 /captures
 .externalNativeBuild
+gradle/wrapper/gradle-wrapper.properties
```

Comment:
@Fafan2008 
Seems we dont really need to ignore them.
Fixing.

Reply:
@Fafan2008 Please, remove gradle folder from gitignore since we need this for travis..